spring-cloud-gcp-docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -26,13 +26,12 @@ dependencies {
 
 === Pub/Sub operations abstraction
 
-`PubSubTemplate` is an abstraction that allows Spring users to use Google Cloud Pub/Sub without
+`PubSubOperations` is an abstraction that allows Spring users to use Google Cloud Pub/Sub without
 depending on any Google Cloud Pub/Sub API semantics.
-`PubSubTemplate` uses the
+It comprehends all the possible operations to be carried by a Google Cloud Pub/Sub client.
```

Comment:
"Comprehends" is an odd choice of words. I also don't think it's completely accurate to say that it covers all possible operations.

Reply:
It sounds natural to me, but English isn't my native language!

How about "It specifies the typical set of operations carried by a Google Cloud Pub/Sub client." ?

Reply:
Nor mine! :)

 "It provides the common set of operations needed for interacting with Google Cloud Pub/Sub.